container/Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM registry.access.redhat.com/ubi8/ubi
 RUN yum -y install java-1.8.0-openjdk
 
-RUN mkdir -p /opt/sling
+ENV MONGO_DB="sling"
+ENV REPOSITORY="sling/repository"
 
-COPY ../pantheon-slingstart/target/pantheon-slingstart-*.jar /opt/sling/pantheon.jar
+RUN mkdir -p /opt/pantheon-karaf-dist
+RUN mkdir /tmp/archive
+WORKDIR /tmp/archive
 
-WORKDIR /opt/sling/
-EXPOSE 8080
-EXPOSE 5005
+COPY ../pantheon-karaf-dist/target/pantheon-karaf-dist-*.tar.gz /tmp/archive/pantheon-karaf-dist.tar.gz
 
-ENV JAVA_OPTS -Xmx1024m -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
-ENV SLING_OPS ''
+RUN tar --strip-components=1 -C /opt/pantheon-karaf-dist -xzf pantheon-karaf-dist.tar.gz; \
+    rm -rf /tmp/*
 
-CMD export COMMIT_HASH=$(cat /opt/sling/commit_hash); java $JAVA_OPTS -jar pantheon.jar $SLING_OPTS
+RUN chmod 755 /opt; 
+    #sed -i "21s/out/stdout/" /opt/pantheon-karaf-dist/etc/org.ops4j.pax.logging.cfg;
+
+EXPOSE 1099 8181 44444
```

Comment:
Why are we exposing  ports : 1099 and 44444 ?

Reply:
They are JMX ports. I will check if there is a different single port of debug and jmx.